test/units/testsuite-17.10.sh

```diff
@@ -9,14 +9,12 @@ set -o pipefail
 # Coverage test for udevadm
 
 cleanup_17_10() {
-        set +e
+    set +e
 
-        losetup -d "$loopdev"
-        rm -f "$blk"
+    losetup -d "$loopdev"
+    rm -f "$blk"
 
-        ip link delete "$netdev"
-
-        return 0
+    ip link delete "$netdev"
 }
```

Comment:
The `return` is intentional. If it's removed, the return value of `ip link delete` will be used as return value of the cleanup function and, transitively, as return value of the whole script. Hence, if the `ip` command fails for whatever reason, the whole `testuite-17.sh` will fail too. I'm pretty sure that's not wanted.

Reply:
In this very specific case it won't - bash's `exit` will restore the "pre-EXIT-trap exit code unless you explicitly call `exit [n]` from the trap handler: https://github.com/bminor/bash/blob/ec8113b9861375e4e17b3307372569d429dec814/builtins/exit.def#L143-L147

i.e.

```
$ bash -c 'trap false EXIT'; echo $?
0
```

but

```
$ bash -c 'trap "exit 1" EXIT'; echo $?
1
```